### PR TITLE
Add Chinese tranlation for docs/python-types.md

### PR DESCRIPTION
**Pull request by waynerv**

### CODE REVIEW

**Review by RunningIkkyu — APPROVED** [review]
Great!

**Comment by tiangolo** [comment]
Thank you @waynerv for your contribution! :rocket: :bow: 

And thanks @RunningIkkyu for the review/approval! :coffee: :cake: 

I'll just wait for an approval review from someone else before merging.

Maybe @Dustyposa or @phy25 , if you have the chance, could you review this? :pray:

File: docs/zh/docs/python-types.md [hunk 1]
```diff
@@ -0,0 +1,287 @@
...
+
+而且添加类型提示一般不会改变原来的运行结果。
+
+现在假设我们又一次正在创建这个函数，这次添加了类型提示。
+
+在同样的地方，你通过 `Ctrl+Space` 触发自动补全，发现：
+
+<img src="https://fastapi.tiangolo.com/img/python-types/image02.png">
+
+这样，你可以滚动查看选项，直到找到正确的"就是它"选项：
```
Comment (phy25) [inline docs/zh/docs/python-types.md:104]: `"就是它"` is weird here.

**Review by phy25 — APPROVED** [review]
Generally fine. Not sure how high the standard will be, but I saw some weird spaces between text and  `“”（）` which doesn't feel quite right.

File: docs/zh/docs/python-types.md (hunk 1 above)
Comment (waynerv) [reply docs/zh/docs/python-types.md]: Fixed all improper spaces I has found, also improved translation of the line above.

**Comment by tiangolo** [comment]
Awesome! Thank you @waynerv ! :cake: :tada: 

And thanks @phy25 for the review :bow: :clap:

**Pull request merged**